jobscript.sh

```diff
@@ -97,4 +99,26 @@ echo "Generating notebook!"
 #shellcheck disable=SC1000-SC9999
 RESULTS_DIR="$(pwd)" SAMPLE_PATH="${sample_filename}" PYTHONWARNINGS="ignore" GIT_DIR="/.git" GIT_WORK_TREE="/" INSTRUMENT_VENDOR="${INSTRUMENT_VENDOR}" ONE_CODEX_REPORT_FILENAME="${ONE_CODEX_REPORT_FILENAME}" ARTIC_PRIMER_VERSION="${ARTIC_PRIMER_VERSION}" conda run -n jobscript-env jupyter nbconvert --execute --to onecodex_pdf --ExecutePreprocessor.timeout=-1 --output="${ONE_CODEX_REPORT_FILENAME}" --output-dir="." report.ipynb
 
+echo "Removing unnecessary files"
+rm -f aa_codes.txt \
+	annot_table.orfs.txt \
+	covid19.bam.bai \
```

Comment:
Please add the index back in. It's easy to generate, true, but keeping it eliminates an extra step if someone wants to look at the data in IGV, and not all users will be immediately familiar with how to generate an index.